checkstyle-suppressions.xml

```diff
@@ -6,4 +6,8 @@
     <suppress checks="Indentation" files="package-info\.java"/>
     <suppress checks="AbbreviationAsWordInName" files="TNT"/>
     <suppress checks="AbbreviationAsWordInName" files="PEBKAC"/>
+
+    <!-- SpongeEventFactory contains autogenerated code which does not obey certain standards. -->
```

Comment:
This should look like this: https://github.com/SpongePowered/SpongeAPI/commit/c1e798256a05c4eccb03c52db19ec62f31916416